src/docker/blockchain/ethereum.rs

```diff
@@ -1,40 +1,43 @@
-use crate::docker::{blockchain::BlockchainImage, ExposedPorts, Image};
+use crate::docker::{ExposedPorts, Image};
+use emerald_rs::PrivateKey;
+use ethabi::Token;
+use futures::compat::Future01CompatExt;
 use lazy_static::lazy_static;
-use tiny_keccak;
+use secp256k1::SecretKey;
+use std::io::Cursor;
+use std::time::Duration;
 use web3::transports::EventLoopHandle;
+use web3::types::{Bytes, TransactionReceipt, H160};
 use web3::{
     api::Web3,
-    futures::Future,
     transports::Http,
-    types::{Address, TransactionRequest, H256, U256},
+    types::{Address, U256},
 };
 
+pub const TOKEN_CONTRACT: &str = include_str!("../../../erc20_token/build/contract.hex");
+pub const CONTRACT_ABI: &str = include_str!("../../../erc20_token/build/abi.json");
+pub const HTTP_URL_KEY: &str = "ETHEREUM_NODE_HTTP_URL";
+
 lazy_static! {
 // expect: Should always be able to parse
-        static ref PARITY_DEV_ACCOUNT: web3::types::Address = "00a329c0648769a73afac7f9381e08fb43dbea72"
-            .parse()
-            .expect("Could not parse DEV account address");
-}
+    static ref DEV_ACCOUNT: web3::types::Address = "00a329c0648769a73afac7f9381e08fb43dbea72"
+        .parse()
+        .expect("Could not parse DEV account address");
 
-pub const HTTP_URL_KEY: &str = "ETHEREUM_NODE_HTTP_URL";
+    static ref DEV_ACCOUNT_PRIVATE_KEY: emerald_rs::PrivateKey = emerald_rs::PrivateKey::try_from(&hex_literal::hex!("4d5db4107d237df6a3d58ee5f70ae63d73d7658d4026f2eefd2f204c81682cb7")).unwrap();
```

Comment:
Use expect instead of unwrap to allow code reader to know why it should not fail